### DISCUSSION TITLE
There is no English documentation of Swoole api and OpenSwoole has now differences at api.

### DISCUSSION THREAD

**Discussion by XDRiVE888** [swoole/swoole-src#4941] [Ideas]
There is no English documentation of swoole api and openswoole has now big differences at api with swoole to use them.

I used to use swoole and swoole.co.uk as a documentation resource. Now the site swoole.co.uk redirects to the site openwoole.com, however, the APIs of these projects are no longer compatible, in particular, in version 22 they stopped supporting MysqliPool and even they did not indicate this in the release.
Thats why i want to use swoole.
  Will the Swoole project's documentation be in English?

**Comment by leocavalcante** [comment]
There are some work going on at https://github.com/swoole/docs to be able to contribute to english documentation.
In the mean while, what I do and recommend is to simple let your browser translate https://wiki.swoole.com
It is not a perfect handmade translation, but it works pretty well.

**Comment by deminy** [comment]
Besides what @leocavalcante has mentioned, you can also use [swoole/ide-helper](https://github.com/swoole/ide-helper) for references. I have been updating it during my spare time for the last few months, and I'm looking forward to finishing documenting it sometime in 2023.
